controllers/runtimecomponent_controller.go

```diff
@@ -61,16 +61,16 @@ type RuntimeComponentReconciler struct {
 	watchNamespaces []string
 }
 
-// +kubebuilder:rbac:groups=rc.app.stacks,resources=runtimecomponents;runtimecomponents/status;runtimecomponents/finalizers,verbs=get;list;watch;create;update;patch;delete,namespace=runtime-component-operator
-// +kubebuilder:rbac:groups=apps,resources=deployments;statefulsets,verbs=get;list;watch;create;update;patch;delete,namespace=runtime-component-operator
+// +kubebuilder:rbac:groups=rc.app.stacks,resources=runtimecomponents;runtimecomponents/status;runtimecomponents/finalizers,verbs=list;watch;create;update;delete,namespace=runtime-component-operator
+// +kubebuilder:rbac:groups=apps,resources=deployments;statefulsets,verbs=list;watch;create;update;delete,namespace=runtime-component-operator
 // +kubebuilder:rbac:groups=apps,resources=deployments/finalizers;statefulsets,verbs=update,namespace=runtime-component-operator
-// +kubebuilder:rbac:groups=core,resources=services;secrets;serviceaccounts;configmaps,verbs=get;list;watch;create;update;patch;delete,namespace=runtime-component-operator
-// +kubebuilder:rbac:groups=autoscaling,resources=horizontalpodautoscalers,verbs=get;list;watch;create;update;patch;delete,namespace=runtime-component-operator
-// +kubebuilder:rbac:groups=networking.k8s.io,resources=ingresses,verbs=get;list;watch;create;update;patch;delete,namespace=runtime-component-operator
-// +kubebuilder:rbac:groups=route.openshift.io,resources=routes;routes/custom-host,verbs=get;list;watch;create;update;patch;delete,namespace=runtime-component-operator
+// +kubebuilder:rbac:groups=core,resources=services;secrets;serviceaccounts,verbs=list;watch;create;update;delete,namespace=runtime-component-operator
```

Comment:
@yongja79 Operator does interact with at least one ConfigMap that has the optional configurations. Relevant controller code is [here](https://github.com/application-stacks/runtime-component-operator/blob/66fe693b8afc7d38b92b68f4a72c14bd7711152a/controllers/runtimecomponent_controller.go#L99)

It only logs the error and proceeds. @arturdzm is that behaviour intentional? Shouldn't the ConfigMap always exist?

Reply:
@leochr I've added back the configMap permissions to the rco-manager role and have also updated the rco-leader-election role